.github/workflows/build-and-push-assets.yml

```diff
@@ -61,7 +81,49 @@ on:
         required: false
 
 jobs:
+  checks:
+    runs-on: ubuntu-latest
+    env:
+      GH_TOKEN: ${{ github.token }}
```

Comment:
For what is this `env.GH_TOKEN` used? Please leave a comment after the assignment for "later". :)